code/game/objects/items/weapons/tools.dm

```diff
@@ -145,6 +146,14 @@
 /obj/item/wirecutters/iswirecutter()
 	return TRUE
 
+/obj/item/wirecutters/bomb
+	name = "bomb defusal wirecutters"
+	desc = "A tool used to delicately sever the wires used in bomb fuses."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "cutters"
+	toolspeed = 0.6
+	bomb_defusal_chance = 90 // 90% chance, because the thrill of dying must be kept at all times, duh
+
```

Comment:
Considering this is pretty much a general use item I'd love to see this as like a briefcase-full-of-tools sort of deal. Like you open it, you can see a lot of tools in the sprite and then you can just defuse things by clicking on bombs with the open briefcase.

Reply:
talked with you about this in discord now, i think we can do this in the future when we [REDACTED] and implement [REDACTED]